src/order/atoms.lean

```diff
@@ -565,21 +565,98 @@ is_simple_order_iff_is_coatom_bot.trans $ and_congr (not_congr subtype.mk_eq_mk)
 
 end set
 
+namespace galois_insertion
```

Comment:
Could you pull out `[partial_order α] [partial_order β]` to the `namespace`?
```suggestion
namespace galois_insertion
variables [partial_order α] [partial_order β]
```
That should unclutter the statements quite a bit.